src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -10,7 +10,8 @@ import _root_.slick.driver.JdbcProfile
 import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
 import cromwell.engine.ExecutionIndex._
 import cromwell.engine.ExecutionStatus._
-import cromwell.engine.backend.{Backend, JobKey, WorkflowQueryResult}
+import cromwell.engine.backend._
+import cromwell.engine.backend.jes.JesBackend
```

Comment:
It wasn't that bad I just added a "key" attribute in the BackendReturnCode class that we can use as a key for the EXECUTION_INFO table